### DISCUSSION TITLE
grasshopper morpho plant2D

### DISCUSSION THREAD

**Discussion by Caidayan** [AntonelloDN/Morpho#28] [Q&A]
Regarding morpho modeling, firstly, how can one modify the facade window materials and roof greening materials on the building form? Secondly, in vegetation modeling, I have set up plant 2D, but I cannot see the vegetation in the space preview. How can these issues be resolved?

**Comment by AntonelloDN** [comment]
Hi @Caidayan,

### Green roofs/walls
Building material component has a input to assign greening materials
![image](https://github.com/AntonelloDN/Morpho/assets/16763953/418325de-522e-49e9-bbd3-afcf1a869a10)

### Plant 2D
There are [rules](https://github.com/AntonelloDN/Morpho/wiki/1.-Grasshopper-plugin#geometry-mapping) for geometries to keep in mind during the modeling phase
